api/routes/states/put.js

```diff
@@ -64,7 +64,12 @@ module.exports = (app, dataHelper = defaultDataHelper) => {
       logger.silly(req, newData);
 
       logger.silly(req, 'validating program benefits');
-      if (joi.string().validate(newData.program_benefits).error) {
+      if (
+        joi
+          .string()
+          .allow('')
```

Comment:
empty string was failing validation, and we want to accept it if user wants to clear out their prior input.

https://github.com/hapijs/joi/issues/448

Reply:
Interesting!